snorkel/learning/gen_learning.py

```diff
@@ -210,11 +168,42 @@ class prior (in log scale)
             L = sparse.csr_matrix(L)
         L = L[idxs, :]
 
+        # Optionally: Remap the values of L (in categorical setting) so that the
```

Comment:
Recommend moving this to another branch and doing user-provided scoping for now. :)